app/controllers/api_controller.rb

```diff
@@ -1,3 +1,8 @@
+#
+# Initializing REST API environment, called once @ startup
+#
+Api::Initializer.new.go
```

Comment:
Ugh, sorry, didn't see this response.  😞   Yeah, I don't know, but it feels like anything that has expectations that it's called once may have issues with code reloading.  Maybe @Fryguy @chrisarcand @bdunne have suggestions.  🙏